### PR TITLE
Removed redundant parsing of VMSnapshot usage record.

### LINKED ISSUE #5662: Usage: Duplicated VMSnapshot usage record
When list usagerecords with type=25 (vm snapshot), got duplicated usage record

```
(localcloud) SBCM5> > list usagerecords type=25 startdate=2021-11-03 enddate=2021-11-03
{
  "count": 10,
  "usagerecord": [
    {
      "account": "admin",
      "accountid": "0e3e52ed-03db-11ec-8879-1e0079000360",
      "description": "VMSnapshot usage Id: 1 (2790057b-5a2f-4f90-ae5b-021573d0943d)  for VM L2 (7a25841b-cfac-49ca-90eb-0cb187ba3414) and size (8.00 GB) 8589934592",
      "domain": "ROOT",
      "domainid": "e5ce2438-03da-11ec-8879-1e0079000360",
      "enddate": "2021-11-03'T'17:11:53+00:00",
      "name": "i-2-1000-VM_VS_20211103170813",
      "rawusage": "0.058435",
      "size": 8589934592,
      "startdate": "2021-11-03'T'17:08:23+00:00",
      "tags": [],
      "usage": "0.058435 Hrs",
      "usageid": "2790057b-5a2f-4f90-ae5b-021573d0943d",
      "usagetype": 25,
      "virtualmachineid": "7a25841b-cfac-49ca-90eb-0cb187ba3414",
      "zoneid": "59260b3d-f587-4a8c-8c58-182ae3f01679"
    },
    {
      "account": "admin",
      "accountid": "0e3e52ed-03db-11ec-8879-1e0079000360",
      "description": "VMSnapshot usage Id: 1 (2790057b-5a2f-4f90-ae5b-021573d0943d)  for VM L2 (7a25841b-cfac-49ca-90eb-0cb187ba3414) and size (8.00 GB) 8589934592",
      "domain": "ROOT",
      "domainid": "e5ce2438-03da-11ec-8879-1e0079000360",
      "enddate": "2021-11-03'T'17:11:53+00:00",
      "name": "i-2-1000-VM_VS_20211103170813",
      "rawusage": "0.058435",
      "size": 8589934592,
      "startdate": "2021-11-03'T'17:08:23+00:00",
      "tags": [],
      "usage": "0.058435 Hrs",
      "usageid": "2790057b-5a2f-4f90-ae5b-021573d0943d",
      "usagetype": 25,
      "virtualmachineid": "7a25841b-cfac-49ca-90eb-0cb187ba3414",
      "zoneid": "59260b3d-f587-4a8c-8c58-182ae3f01679"
    },


```

in /var/log/cloudstack/usage/usage.log, it also shows usage records are created twice.

```
# grep "Creating VMSnapshot" /var/log/cloudstack/usage/usage.log
2021-11-03 17:12:15,076 DEBUG [usage.parser.VMSnapshotUsageParser] (Usage-Job-1:null) (logid:) Creating VMSnapshot Id:1 Volume usage record for vol: 996, usage: 0.058435, startDate: Wed Nov 03 17:08:23 UTC 2021, endDate: Wed Nov 03 17:11:53 UTC 2021, for account: 2
2021-11-03 17:12:15,085 DEBUG [usage.parser.VMSnapshotUsageParser] (Usage-Job-1:null) (logid:) Creating VMSnapshot Id:1 Volume usage record for vol: 996, usage: 0.058435, startDate: Wed Nov 03 17:08:23 UTC 2021, endDate: Wed Nov 03 17:11:53 UTC 2021, for account: 2
2021-11-03 17:12:15,094 DEBUG [usage.parser.VMSanpshotOnPrimaryParser] (Usage-Job-1:null) (logid:) Creating VMSnapshot Id: 1 On Primary usage record for vm: 1000, usage: 0.058435, startDate: Wed Nov 03 17:08:23 UTC 2021, endDate: Wed Nov 03 17:11:53 UTC 2021, for account: 2
2021-11-03 17:17:14,379 DEBUG [usage.parser.VMSnapshotUsageParser] (Usage-Job-1:null) (logid:) Creating VMSnapshot Id:1 Volume usage record for vol: 996, usage: 0.083334, startDate: Wed Nov 03 17:11:53 UTC 2021, endDate: Wed Nov 03 17:16:53 UTC 2021, for account: 2
2021-11-03 17:17:14,384 DEBUG [usage.parser.VMSnapshotUsageParser] (Usage-Job-1:null) (logid:) Creating VMSnapshot Id:1 Volume usage record for vol: 996, usage: 0.083334, startDate: Wed Nov 03 17:11:53 UTC 2021, endDate: Wed Nov 03 17:16:53 UTC 2021, for account: 2
2021-11-03 17:17:14,389 DEBUG [usage.parser.VMSanpshotOnPrimaryParser] (Usage-Job-1:null) (logid:) Creating VMSnapshot Id: 1 On Primary usage record for vm: 1000, usage: 0.083334, startDate: Wed Nov 03 17:11:53 UTC 2021, endDate: Wed Nov 03 17:16:53 UTC 2021, for account: 2

```

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
Usage
~~~

##### CLOUDSTACK VERSION

~~~
4.14, 4.15, 4.16
~~~

##### CONFIGURATION

N/A

##### OS / ENVIRONMENT

N/A

##### SUMMARY

CloudStack usage server generated duplicated usage record for vmsnapshot

##### STEPS TO REPRODUCE

~~~
1. change global settings for usage server , restart cloudstack management and usage server

enable.usage.server = true
usage.stats.job.aggregation.range = 5

2. create vm;
3. create vm snapshot
4. wait for 5-10 minutes, check usagerecord by cmk or cloud_usage.cloud_usage table (usage type=25).
~~~

##### EXPECTED RESULTS

~~~
only 1 usage record for the vm snapshot.
~~~

##### ACTUAL RESULTS

~~~
duplicated usage records.
~~~

### PR DESCRIPTION
**Pull request by sureshanaparti**
### Description

This PR removes the redundant parsing of VMSnapshot usage record.

Fixes #5662 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [x] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: usage/src/main/java/com/cloud/usage/UsageManagerImpl.java [hunk 1]
```diff
@@ -958,13 +958,7 @@ private boolean parseHelperTables(AccountVO account, Date currentStartDate, Date
                 s_logger.debug("VM Snapshot usage successfully parsed? " + parsed + " (for account: " + account.getAccountName() + ", id: " + account.getId() + ")");
             }
         }
```
Comment (sureshanaparti) [inline usage/src/main/java/com/cloud/usage/UsageManagerImpl.java:960]: same parsing is performed at line# 955 - 960.

**Comment by sureshanaparti** [comment]
@blueorangutan package

**Review by DaanHoogland — APPROVED** [review]
clgtm (no real functional changes)

**Comment by sureshanaparti** [comment]
@blueorangutan package

**Comment by sureshanaparti** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test keepEnv

**Comment by DaanHoogland** [comment]
tested in a lab, works as expected.

**Review by borisstoyanov — APPROVED** [review]
LGTM based on code review

**Pull request merged**